docs/sources/installation/helm/monitor-and-alert/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Configure monitoring and alerting
+menuTitle: Configure monitoring and alerting
+description: setup monitoring and alerts for the Helm Chart
+aliases:
+  - /docs/writers-toolkit/latest/templates/task-template
+weight: 100
+keywords:
+  - monitoring 
+  - alert
+  - alerting
+---
+
+# Configure monitoring and alerting
+
+By default this Helm Chart configures meta-monitoring of metrics (service monitoring) and logs (self monitoring).
+
+The `ServiceMonitor` resource works with either the Prometheus Operator or the Grafana Agent Operator, and defines how Loki's metrics should be scraped.
+Scraping this Loki cluster using the scrape config defined in the `SerivceMonitor` resource is required for the included dashboards to work.
+
+*Self monitoring* is enabled by default. This will deploy a `GrafanaAgent`, `LogsInstance`, and `PodLogs` resource which will instruct the Grafana Agent Operator (installed seperately) on how to scrape this Loki cluster's logs and send them back to itself. Scraping this Loki cluster using the scrape config defined in the `PodLogs` resource is required for the included dashboards to work.
+
+Rules and alerts are automatically deployed.
+
+**Before you begin:**
+
+- Helm 3 or above. See [Installing Helm](https://helm.sh/docs/intro/install/).
+- A running Kubernetes cluster with a running Loki deployment.
+- A running Grafana instance.
+
+**To install the dashboards:**
+
+1. Enable dashboards with one of these methods:
+
+    - Modify the `values.yaml`:
+
+      ```yaml
+      monitoring:
+        dashboards:
+          enabled: false
+      ``` 
+
+    - Use the command line parameter:
+
+      ```bash
+      helm upgrade --set monitoring.dashboards.enabled = true loki grafana/loki
+      ```
+
+1. Set `monitoring.dashboards.namespace` to the namespace of the Grafana instance if it is in a different namespace than this Loki cluster.
+
+**To use the Grafana Agent for service monitoring:**
+
+TODO: Is this possible?
+
+**To add add additional Prometheus rules:**
```

Comment:
@trevorwhitney what is the value of this?

Reply:
in other words, why would someone want to add custom recording rules? That's a good question. Assuming they have other means to add these rules, which they would, someone would only want to add rules here as they pertain to loki metrics. So, if someone has some customer dashboards around loki itself they might want to add rules here, but you're right for the majority of users using our dashboards this likely isn't necessary. Should we remove these values from the helm chart?